cognite/client/data_classes/data_modeling/instances.py

```diff
@@ -259,7 +256,7 @@ def __init__(
         self,
         space: str,
         external_id: str,
-        version: str,
+        version: int,
```

Comment:
Is this a recent change, or a bug in the original implementation?

Reply:
bug in the original.